internal/app/handler/get_ping.go

```diff
@@ -0,0 +1,34 @@
+package handler
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/FoGezz/go-url-shortener/cmd/shortener/config"
+)
+
+type getPingHandler struct {
+	ShortenerHandler
+}
+
+func NewGetPingHandler(app *config.App) *getPingHandler {
```

Comment:
Why do you need it?

Reply:
just have the same form as another handlers